### PR TITLE
[staking] set fairbank height to around June 01 3pm

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: blockchain/genesis/genesis.go [hunk 1]
```diff
@@ -55,16 +55,16 @@ func defaultConfig() Genesis {
...
+			FairbankBlockHeight:     5165641,
 		},
 		Account: Account{
 			InitBalanceMap: make(map[string]string),
 		},
 		Poll: Poll{
 			PollMode:                         "nativeMix",
 			EnableGravityChainVoting:         true,
-			GravityChainCeilingHeight:        19937937, // TODO: replace with accurate height before Fairbank launch
+			GravityChainCeilingHeight:        10199000,
```
Comment (dustinxie) [inline blockchain/genesis/genesis.go:67]: average ether block time is 13second: https://etherscan.io/chart/blocktime
which gives 10193315 at June 02 2pm

use 12.5 second block time --> 10199000 for some extra margin

**Comment by yutongp** [comment]
also need to set foundationBonusP2StartEpoch, FoundationBonusP2EndEpoch

File: blockchain/genesis/genesis.go [hunk 2]
```diff
@@ -55,16 +55,16 @@ func defaultConfig() Genesis {
 			DardanellesBlockHeight:  1816201,
 			DaytonaBlockHeight:      3238921,
 			EasterBlockHeight:       4478761,
-			FbkMigrationBlockHeight: 4330441,
-			FairbankBlockHeight:     4339081,
+			FbkMigrationBlockHeight: 5159881,
```
Comment (yutongp) [inline blockchain/genesis/genesis.go:58]: this can be even earlier. it doesn't hurt. maybe 10 hours earlier than fb height

Comment (dustinxie) [reply blockchain/genesis/genesis.go:58]: set to 12 hrs before

**Pull request merged**